docs/sql-data-sources-orc.md

```diff
@@ -31,7 +31,7 @@ the vectorized reader is used when `spark.sql.hive.convertMetastoreOrc` is also
   <tr>
     <td><code>spark.sql.orc.impl</code></td>
     <td><code>native</code></td>
-    <td>The name of ORC implementation. It can be one of <code>native</code> and <code>hive</code>. <code>native</code> means the native ORC support that is built on Apache ORC 1.4. `hive` means the ORC library in Hive 1.2.1.</td>
+    <td>The name of ORC implementation. It can be one of <code>native</code> and <code>hive</code>. <code>native</code> means the native ORC support that is built on Apache ORC 1.5. `hive` means the ORC library in Hive 1.2.1.</td>
```

Comment:
Let's just remove versions from this statement entirely? it's not the right place

Reply:
+1 for removal.

Reply:
Ur, BTW, it seems that we need to rewrite all. `Hive 1.2.1` also became wrong because `Hadoop 3.2` profile means ORC 1.6, too.

cc @wangyum